x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -65,17 +66,26 @@ const getColumnConfigs = ({
         defaultMessage: 'Name',
       }),
       order: 10,
-      render: (index) => (
-        <>
-          <EuiLink
-            data-test-subj="indexTableIndexNameLink"
-            onClick={() => history.push(getIndexDetailsLink(index.name, location.search || ''))}
-          >
-            {index.name}
-          </EuiLink>
-          {renderBadges(index, extensionsService, filterChanged)}
-        </>
-      ),
+      render: (index) => {
+        return (
+          <>
+            <EuiLink
+              data-test-subj="indexTableIndexNameLink"
+              onClick={() => {
+                if (!extensionsService.indexDetailsPageRoute) {
+                  history.push(getIndexDetailsLink(index.name, location.search || ''));
+                } else {
+                  const url = extensionsService.indexDetailsPageRoute.renderRoute({ index });
+                  application.navigateToUrl(`${url}${index.name}`);
```

Comment:
```suggestion
                  const url = extensionsService.indexDetailsPageRoute.renderRoute({ indexName: index.name });
                  application.navigateToUrl(url);
```

Reply:
updated in https://github.com/elastic/kibana/pull/194857/commits/9ab2e9b23640da390cbe2f8dfb7e3e7ced3badf6